src/lib/y2packager/product.rb

```diff
@@ -47,6 +47,15 @@ def self.available_base_products
       Y2Packager::ProductReader.available_base_products
     end
 
+    # Returns the selected base product
+    #
+    # It assumes that at most 1 product could be selected.
+    #
+    # @return [Y2Packager::Product] Selected base product
+    def self.selected_base
+      available_base_products.find(&:selected?)
```

Comment:
Yes, I think that this class will evolve. We will adapt the API (or split into new classes) as soon as we need to do that.